dotnet/samples/Hello/HelloAIAgents/HelloAIAgent.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AutoGen.Abstractions;
+using Microsoft.AutoGen.Agents;
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Hello;
+[TopicSubscription("HelloAgents")]
+public class HelloAIAgent(
+    IAgentContext context,
+    [FromKeyedServices("EventTypes")] EventTypes typeRegistry,
+    IChatClient client) : HelloAgent(
+        context,
+        typeRegistry),
+        ISayHello,
```

Comment:
Nit: i think the inheritance is redundant since `HelloAgent` Is already a `ISayHello`